### PR TITLE
Release 2.0.7

### LINKED ISSUE #649: Rebuild against latest public-transport-enabler to get Sweden fix
Hi!

Sweden stopped working a while ago. The issue has been fixed in public-transport-enabler (see schildbach/public-transport-enabler#266).

Is there any chance you guys could trigger a rebuild of the app so we can get that fix?

Thanks!

### PR DESCRIPTION
**Pull request by ialokim**
As requested in https://github.com/grote/Transportr/issues/649#issuecomment-562136699, I've prepared the next release.

I've updated the libraries, bumped the version number and tested the release. However, I'm not able to update translations as I don't have the necessary permissions at Transifex.

Fixes #649

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -105,14 +105,14 @@ dependencies {
...
+	implementation 'androidx.recyclerview:recyclerview:1.1.0'
 	implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
 	implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
 	implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
 	implementation 'androidx.lifecycle:lifecycle-common-java8:2.1.0'
-	implementation 'androidx.room:room-runtime:2.2.0'
-	kapt 'androidx.room:room-compiler:2.2.0'
+	implementation 'androidx.room:room-runtime:2.2.2'
+	kapt 'androidx.room:room-compiler:2.2.2'
```
Comment (grote) [inline app/build.gradle]: Normally, I update non-test libs only short after a release, but not short before to not introduce new bugs. PTE is an exception to that rule.

File: app/src/main/res/drawable/network_czechia_logo.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="900dp"
-    android:height="600dp"
+    android:width="64dp"
+    android:height="64dp"
```
Comment (grote) [inline app/src/main/res/drawable/network_czechia_logo.xml]: Nice catch!

File: fastlane/metadata/android/en-US/changelogs/116.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+* Added data source for Czechia (Thanks Ondra!)
+* Display standard fares for trip results
+* Better error handling in case of failing trip queries
```
Comment (grote) [inline fastlane/metadata/android/en-US/changelogs/116.txt]: Thanks for thinking of the changelog.

We should maybe document this and the entire release process somewhere. Normally, the changelog is first added to [`src/main/res/xml/changelog_master.xml`](https://github.com/grote/Transportr/blob/master/app/src/main/res/xml/changelog_master.xml) to appear in the UI after updates. Then, [`fastlane/generate_changelog.py`](https://github.com/grote/Transportr/blob/master/fastlane/generate_changelog.py) is run to produce the Google play changelog.

Comment (grote) [reply fastlane/metadata/android/en-US/changelogs/116.txt]: I think the two lines above are missing a (Thanks Mikolai!) ;)

**Comment by grote** [comment]
The translations get updated by calling `tx pull --mode=developer -a` in the root folder of the project (where the `.tx` folder is). When new languages are added, they need to get added to the two language arrays in [`app/src/main/res/values/arrays.xml`](https://github.com/grote/Transportr/blob/master/app/src/main/res/values/arrays.xml#L16) and ideally also the changelog. Bonus points for updating translators [in the contributors file](https://github.com/grote/Transportr/blob/master/app/src/main/java/de/grobox/transportr/about/Contributors.kt#L58).

File: app/build.gradle (hunk 1 above)
Comment (ialokim) [reply app/build.gradle]: I've changed it now accordingly to update only PTE and test libraries.

File: fastlane/metadata/android/en-US/changelogs/116.txt (hunk 3 above)
Comment (ialokim) [reply fastlane/metadata/android/en-US/changelogs/116.txt]: Done :)

File: app/build.gradle (hunk 1 above)
Comment (ialokim) [reply app/build.gradle]: I've changed it now accordingly to update only PTE and test libraries.

**Comment by ialokim** [comment]
I've incorporated your comments and added the new translations. This should be ready now. :)

**Comment by grote** [comment]
Awesome, many thanks!

Looking forward to the documentation of the release process ;)

**Comment by ahstro** [comment]
Sorry, just wanted to check if you've "done the F-Droid stuff"? 😇 Haven't seen an update, and I'm hoping for this to fix the issue with it not working in Sweden.

**Comment by ialokim** [comment]
> Sorry, just wanted to check if you've "done the F-Droid stuff"? innocent Haven't seen an update, and I'm hoping for this to fix the issue with it not working in Sweden.

Unfortunately, we've encountered a bug with the new 2.0.7 release (see #652 and #653), so this is why the release has been delayed. We hope to have it ready soon!

**Pull request merged**